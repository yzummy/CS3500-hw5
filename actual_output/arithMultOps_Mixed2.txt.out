
___Entering new scope...

TOKEN: LPAREN 		 LEXEME: (
TOKEN: FLOATCONST 		 LEXEME: 2.71
CONST -> FLOATCONST
FACTOR -> CONST
TOKEN: DIV 		 LEXEME: /
MULT_OP -> /
TOKEN: TRUE 		 LEXEME: TRUE
CONST -> TRUE
FACTOR -> CONST
TOKEN: RPAREN 		 LEXEME: )
MULT_OP_LIST -> epsilon
MULT_OP_LIST -> MULT_OP FACTOR MULT_OP_LIST
HEre 1
1 and: TRUE and

TERM -> FACTOR MULT_OP_LIST
HERE 1111
ADD_OP_LIST -> epsilon
SIMPLE_ARITHLOGIC -> TERM ADD_OP_LIST
ARITHLOGIC_EXPR -> SIMPLE_ARITHLOGIC
EXPR -> ARITHLOGIC_EXPR
FACTOR -> ( EXPR )
TOKEN: POWER 		 LEXEME: ^
MULT_OP -> ^
TOKEN: INTCONST 		 LEXEME: 2
CONST -> INTCONST
FACTOR -> CONST
MULT_OP_LIST -> epsilon
MULT_OP_LIST -> MULT_OP FACTOR MULT_OP_LIST
HEre 1
1 and: FALSE and

TERM -> FACTOR MULT_OP_LIST
HERE 1111
ADD_OP_LIST -> epsilon
SIMPLE_ARITHLOGIC -> TERM ADD_OP_LIST
ARITHLOGIC_EXPR -> SIMPLE_ARITHLOGIC
EXPR -> ARITHLOGIC_EXPR
START -> EXPR

---- Completed parsing ----

Value of the expression is: 7.34

___Exiting scope...

