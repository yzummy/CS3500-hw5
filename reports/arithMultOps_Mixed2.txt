
---- Completed parsing ----					---- Completed parsing ----

 Value of the expression is: 0.00			      |	Value of the expression is:   7.34
