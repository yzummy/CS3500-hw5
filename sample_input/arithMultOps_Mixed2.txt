(2.71 / TRUE) ^ 2
