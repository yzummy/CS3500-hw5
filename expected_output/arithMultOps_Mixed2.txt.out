
---- Completed parsing ----

Value of the expression is:   7.34
